File: aulaMVCfixacao/aulaMVCfixacao/ViewController.m
//
//  ViewController.m
//  aulaMVCfixacao
//
//  Created by Usuário Convidado on 25/03/19.
//  Copyright © 2019 Paulera. All rights reserved.
//

#import "ViewController.h"
#import "Jogador.h"

@interface ViewController ()

@end

@implementation ViewController


- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}
- (IBAction)btnLogar:(id)sender {
    Jogador *jog1 = [[Jogador alloc] init];
    jog1.login = self.txtLogin.text;
    [jog1 logar];
   
    
}

- (IBAction)txtComprar:(id)sender {
    Jogador *jog1 = [[Jogador alloc] init];
    jog1.ponto = self.txtPonto.text.intValue;
    self.txtPonto.text =[NSString stringWithFormat:@"%d",[jog1 comprarComValor:100]];
    
}


@end

File: TreinandoClassesSwift/TreinandoClassesSwift/ViewController.swift
//
//  ViewController.swift
//  TreinandoClassesSwift
//
//  Created by Usuário Convidado on 07/06/19.
//  Copyright © 2019 Paulera. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBOutlet weak var txtNome: UITextField!
    
    @IBOutlet weak var txtMesa: UITextField!
    @IBAction func sldHorario(_ sender: Any) {
    }
    
    @IBOutlet weak var lblHorario: UILabel!
    
    @IBAction func stpqtd(_ sender: Any) {
    }
    @IBOutlet weak var lblqtd: UILabel!
    
    
    @IBOutlet weak var sgcTipo: UISegmentedControl!
    
    
    @IBOutlet weak var swtHiver: UISwitch!
    
    @IBAction func btnPassar(_ sender: Any) {
    }
    
}

